lib/ttfunk/table/name.rb

```diff
@@ -88,55 +142,64 @@ def postscript_name
 
       def parse!
         count, string_offset = read(6, 'x2n*')
+        entries = []
 
-        @entries = []
```

Comment:
There's a leftover `attr_reader` for this. Either remove that or keep this an ivar.

Related question: should it stay available for reading?